src/main/java/io/github/genomicdatainfrastructure/daam/services/SubmitApplicationService.java

```diff
@@ -25,7 +25,7 @@ public class SubmitApplicationService {
     private final RemsApplicationCommandApi remsApplicationCommandApi;
     private final RemsApplicationsApi remsApplicationsApi;
 
-    private static final Set<ApplicationStateEnum> STATES_FORBIDDEN_FOR_SUBMISSION = Set.of(ApplicationStateEnum.DRAFT, ApplicationStateEnum.RETURNED);
+    private static final Set<String> VALID_STATES_FOR_SUBMISSION = Set.of(ApplicationStateEnum.DRAFT.toString(), ApplicationStateEnum.RETURNED.toString());
```

Comment:
important: please move it back to `ApplicationStateEnum`, unless there is a need to transform it to string.